src/pretix/plugins/banktransfer/payment.py

```diff
@@ -234,3 +235,21 @@ def shred_payment_info(self, obj):
         d['_shredded'] = True
         obj.info = json.dumps(d)
         obj.save(update_fields=['info'])
+
+    def payment_refund_supported(self, payment: OrderPayment) -> bool:
+        return all(payment.info_data.get(key) for key in ("payer", "iban", "bic"))
+
+    def payment_partial_refund_supported(self, payment: OrderPayment) -> bool:
+        return self.payment_refund_supported(payment)
+
+    def execute_refund(self, refund: OrderRefund):
+        """
+        We just keep a created refund object. It will be marked as done using the control view
+        for bank transfer refunds.
+        """
+        if refund.payment:
+            refund.info_data = {key: value for key, value in refund.payment.info_data.items() if key in {"payer", "iban", "bic"}}
+            refund.save(update_fields=["info"])
```

Comment:
… and else? We should better throw an exception, or is there a valid use case of the else branch?